lib/WorseReflection/Core/DefaultResolverFactory.php

```diff
@@ -112,4 +118,15 @@ public function createResolvers(): array
             AssignmentExpression::class => new AssignmentExpressionResolver(),
         ];
     }
+
+    private function createStubRegistry(): FunctionStubRegistry
+    {
+        return new FunctionStubRegistry([
+            'array_sum' => new ArraySumStub(),
+            'is_null' => new IsSomethingStub(TypeFactory::null()),
+            'is_float' => new IsSomethingStub(TypeFactory::float()),
+            'is_int' => new IsSomethingStub(TypeFactory::int()),
+            'is_string' => new IsSomethingStub(TypeFactory::string()),
```

Comment:
can make this an extension point in the future, although these could also be implemented with `@phpactor-type-assert` or something, aliasing to Psalm's asserts.